### ISSUE TITLE
Idea: Music player in the menu

### ISSUE THREAD

**Issue by VincentAraneo** [SuperTux/supertux#2900]
An option to add any music from the game (including custom music) in the menu, even if the customization in the menu is bigger, since we have to change the menu screen for a level. And why not change the music on the menu in your own way? that's the function of the Music Player!

What does it do and how can I find and use it?

basically to put a song in the menu, go to the settings, then to the extra section, and you will find an option that will be for the Music Player. the option is called *Change Menu Music*. It will then direct you to the Musics folder just like in the editor when you put in a music. Then select any song that is .music and you're done. That's the idea of how to find the option

**Comment by mrkubax10** [comment]
Unnecessary overcomplication, just mute sound in game and play desired music through any media player.

**Comment by Vankata453** [comment]
> Unnecessary overcomplication, just mute sound in game and play desired music through any media player.

It also allows for ruining ambience in general. There is a reason why there is different music for many environments and moods.

If you still wish to play custom music, you do that.

**Comment by Eauix** [comment]
I wouldn't say this can't be useful for some people (like me, who gets pretty frustated having to hear "Daaaah- Dadadadadadaaaah deedeeeeeeeeeeeeeh" every single time I open the game to test the custom stuff I make😶).

I do often find myself deleting the "theme.ogg" file whenever I install a new version of the game, so, I'd approve this idea :P

**Comment by tylerandari13** [comment]
> I wouldn't say this can't be useful for some people (like me, who gets pretty frustated having to hear "Daaaah- Dadadadadadaaaah deedeeeeeeeeeeeeeh" every single time I open the game to test the custom stuff I make😶).
> 
> I do often find myself deleting the "theme.ogg" file whenever I install a new version of the game, so, I'd approve this idea :P

I agree with this, it wouldnt take anything away to have something like this in ST, but it would probably be on low priority as at the same time it doesnt add much either.

**Comment by Hypernoot** [comment]
Actually, you can replace the menu level by any other level by yourself. And you can put there any music you want.
